fdpa-resolutions.html

```diff
@@ -86,5 +28,159 @@ <h2>Fair Districts PA - Local Resolutions</h2>
+              zoomControlOptions: {
+              position: google.maps.ControlPosition.TOP_LEFT
+          },
+          fullscreenControl: true,
+          streetViewControl: false,
+          mapTypeControl: false
+        });
+        var style = [
+          {
+            featureType: 'all',
+            elementType: 'all',
+            stylers: [
+              { saturation: -85 }
+            ]
+          },
+          {
+            featureType: 'poi',
+            elementType: 'all',
+            stylers: [
+              { visibility: 'off' }
+            ]
+          }
+        ];
+        var styledMapType = new google.maps.StyledMapType(style, {
+          map: map,
+          name: 'Styled Map'
+        });
+        map.mapTypes.set('map-style', styledMapType);
+        map.setMapTypeId('map-style');
+        layer_0 = new google.maps.FusionTablesLayer({
+          query: {
+            select: "col12>>0",
+            from: "1emKZpIHeBPHfcWjdckvWJ_DWmgNrJYTRAMrmllzJ",
+            where: " col3>>0 = 'County'"
+          },
+          map: map,
+          styleId: 3,
+          templateId: 4
+        });
+        layer_1 = new google.maps.FusionTablesLayer({
+          query: {
+            select: "col16>>0",
+            from: "1emKZpIHeBPHfcWjdckvWJ_DWmgNrJYTRAMrmllzJ",
+            where: " col3>>0 in ('Borough', 'City', 'Town', 'Township (1st)', 'Township (2nd)', 'Organization') and col5>>0 in ('In work', 'Materials sent', 'Passed')"
+          },
+          map: map,
+          styleId: 4,
+          templateId: 5
+        });
+      }
+      (function() {
+        var apiKey = {
+          amyprime: 'AIzaSyBxdPyKNXkcFbFPs5I_e-xqGCUPGVuyabs',
+          devel: 'AIzaSyBz3CQw1aK0S9UHwxKq-YZtIeqdcOby-Jc'
+        };
+        var locationUrl = new URL(window.location.href);
+
+        addGoogleMaps();
+
+        var statuses = ['passed', 'inWork'];
+        var countElIds = {
+          fallbackCount: 'fall-back-count',
+          currentCount: 'current-count'
+        };
+        var windowPromise =
+          !!document.getElementById(countElIds.fallbackCount) ?
+          Promise.resolve(true) :
+          new Promise(function(windowLoaded) {
+            window.addEventListener('load', windowLoaded);
+          });
+
+        new Promise(function(scriptAdded){
+          addScript(isProduction() ?
+            'https://cdn.rawgit.com/RobinIsTheBird/fdpa-fusiontables/master/distribution/fusionApi_es5.js' :
+            'https://rawgit.com/RobinIsTheBird/fdpa-fusiontables/master/distribution/fusionApi_es5.js',
+            scriptAdded);
+        }).then(function(){
+          var promisedCounts = statuses.map(function(statusSought) {
+            return new Promise(function(gotCount){ // No error handler
+              fdpaResolutions.getResolutionsCount({
+                status: statusSought,
+                fusiontable: '10Uc_t_dBYUV_K_j6HdCMSgLXGs94bXvunQBGDOjF',
+                googleApiKey: getKey(),

```

Comment:
I'm not positive I'm doing it right, but I'm trying to test with my original API key, and I can see the map, but for the counts I get: "The request did not specify any referer. Please ensure that the client is sending referer or use the API Console to remove the referer restrictions."

Reply:
I sent you email with a long explanation. Cutting to the quick here, I took the liberty to correct your API key restrictions and now it works.

Reply:
Hooray!